Markup/cloud-mgt-app/001_Landing_page.html

```diff
@@ -19,44 +18,79 @@
     <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+    <![endif]-->
```

Comment:
Duplicate `<![endif]-->`. Please verify if its still there and applicable.

Reply:
Fixed